CHANGES.rst

```diff
@@ -65,6 +65,8 @@ Improvements
   reviewing
 - Show a nicer error message when entering an excessively high base
   registration fee (:issue:`3260`)
+- Make link color handling for LaTeX pdfs configurable (thanks
```

Comment:
can you add a link to the PR as well (like in other changelog entries)?

Reply:
Done